rpc/models.py

```diff
@@ -584,3 +581,28 @@ def __str__(self):
             self.by,
             self.time.strftime("%Y-%m-%d"),
         )
+
+
+class ApprovalLogMessage(models.Model):
+    """Public log of final approval-related steps
+
+    These messages will be displayed on the approvals
+    views (historically the AUTH48 approvals page) and
+    will be publically visible.
+    """
+
+    rfc_to_be = models.ForeignKey(RfcToBe, on_delete=models.PROTECT)
```

Comment:
Intentional to let an ApprovalLogMessage block deletion of an RfcToBe? I'd have expected it to be the other way around with other safeguards preventing such deletion, which should be extremely rare.

(This is not an objection, just checking - it should be rare enough that maybe the admin having to explicitly remove the log messages before doing the delete is useful as a safeguard)

Reply:
I didn't put a lot of thinking into this - I just took RpcComment's approach.